### DISCUSSION TITLE
Problem in retraining spacy for sentence splitting

### DISCUSSION THREAD

**Discussion by nikhilajoshy** [explosion/spaCy#12573] [Help: Other Questions] [answered]
I created training data using 
```
for _, rw in tqdm(data.iterrows()): 
    doc = nlp.make_doc(rw.text)
    for token_id, token in enumerate(nlp(rw.text)):
        if token.idx in rw['starts']:
            doc[token_id].is_sent_start = True

        else:
            doc[token_id].is_sent_start = False
    for sent in doc.sents:
        print("--", sent, "---")
    db.add(doc)

db.to_disk("./train.spacy")
```

But when I debug config file, I am getting this
![15b51f23-7851-4be7-8e1c-da2c26dd4f18](https://user-images.githubusercontent.com/37141775/234191960-d4628e00-5ab5-4a3a-95b3-39e9bee3f95f.jpg)

why is training data split as one word per sentence??

**Comment by adrianeboyd** [comment, marked as answer]
For data where you have set sentence boundaries with `token.is_sent_start`, you want to train a `senter` component instead of a `parser` component.

(The `parser` uses dependency trees to identify sentence boundaries, so in data without dependency annotation, each word looks like its own separate tree and own separate sentence.)
